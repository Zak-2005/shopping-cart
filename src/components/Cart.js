import { Outlet } from "react-router-dom";
import { useState } from "react";

const Cart = ({itemsInCart}) =>{
    return (
        <div>
        <h1>Cart {itemsInCart}</h1>
        </div>
    )
}

export default Cart;