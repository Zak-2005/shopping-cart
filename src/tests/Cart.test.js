import React from "react";
import { render } from "@testing-library/react";
import Cart from "../components/Cart";
import "@testing-library/jest-dom";
import { HashRouter } from "react-router-dom";

describe("Cart Component", () => {
  it("Renders Cart text and items", () => {
    
    const { container } = render(<HashRouter><Cart itemsInCart={0}/></HashRouter>);
    expect(container).toMatchSnapshot();
    
  });
});
