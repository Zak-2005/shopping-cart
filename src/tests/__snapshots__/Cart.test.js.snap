// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Cart Component Renders Cart text and items 1`] = `
<div>
  <div
    class="Cart"
  >
    <h1>
      Cart Quantity: 
      0
    </h1>
  </div>
</div>
`;
